helm/starboard-exporter/values.yaml

```diff
@@ -84,3 +84,10 @@ networkpolicy:
   enabled: true
 
 podAnnotations: {}
+
+customMetricsHpa:
```

Comment:
Total nitpick but it would be nice to align this style with other acronyms.
```suggestion
customMetricsHPA:
```

Reply:
Fixed